Source/WebCore/platform/graphics/mac/controls/ControlMac.mm

```diff
@@ -43,6 +47,11 @@
 {
 }
 
+bool ControlMac::userPrefersContrast()
+{
+    return [[NSWorkspace sharedWorkspace] accessibilityDisplayShouldIncreaseContrast];
```

Comment:
Which process does this run in? If it's in the GPU Process, we should feed it in from the web process.

Reply:
I will address this issue in a separate patch.